main/SQLHelper.cpp

```diff
@@ -3587,6 +3587,9 @@ uint64_t CSQLHelper::CreateDevice(const int HardwareID, const int SensorType, co
 		case sTypeCounterIncremental:		//Counter Incremental
 			DeviceRowIdx = UpdateValue(HardwareID, ID, 1, SensorType, SensorSubType, 12, 255, 0, "0", devname);
 			break;
+		case sTypeManagedCounter:		//Counter Incremental
```

Comment:
I think the comment here is wrong, could be removed (also for the counter incremental if you would)